wye/profiles/forms.py

```diff
@@ -47,3 +47,14 @@ def __init__(self, *args, **kwargs):
     class Meta:
         model = models.Profile
         exclude = ('user', 'slug')
+
+
+class ContactUsForm(forms.Form):
+
+    name = forms.CharField(label='Your name*', required=True)
+    email = forms.EmailField(label='Your email*', required=True)
+    content = forms.CharField(label='Your message*', required=True, widget=forms.Textarea)
+    mobile_number = forms.CharField(label='Your contact number', required=False)
+    query_topic = forms.ChoiceField(label='Select Option*', required=True,
```

Comment:
Feedback_topic. Can we define constant like in base.constanst with value
1) Workshop (2) Account (3) Organisation (4) Others